html/style.css

```diff
@@ -248,14 +248,58 @@ input.inline {
   margin-left: 0.5em;
 }
 
+/* Console fullscreen */
+.fullscreen-box {
+  position: relative;
+}
+
+.fullscreen-box.full {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  left: 0;
+  top: 0;
+  z-index: 1001;
+}
+
+.fullscreen-box > .fullscreen-control {
+  position: absolute;
+  border-radius: 2px;
+  top: 16px;
+  right: 16px;
+  width: 1em;
+  height: 1em;
+  text-align: center;
+  background: #00000025;
+  padding: 15px;
+  line-height: 1em;
+  color: #fff;
+  cursor: pointer;
+}
+
+.fullscreen-box > .fullscreen-control::before {
+  content: "F";
+}
+
+.fullscreen-box.full > .fullscreen-control::before {
+  content: "X";
+}
+
+.fullscreen-box > pre.console {
+  height: 500px;
+}
+
+.fullscreen-box.full > pre.console {
+  height: 100vh;
+}
+
 /* Text console */
 pre.console {
   background-color: #663300;
   border-radius: 5px;
   border: 0px solid #000000;
   color: #66ff66;
   padding: 5px;
-  overflow: scroll;
```

Comment:
what is the effect of this on the regular console?

Reply:
It repeats this style, delete this is no matter.
https://github.com/jeelabs/esp-link/blob/master/html/pure.css#L227-L233